### ISSUE TITLE
Glob support for ** using `mattn/go-zglob`

### ISSUE THREAD

**Issue by dkyeremeh** [drone-plugins/drone-gitea-release#40]
### TLDR;
I'm trying to use `drone-gitea-release` to publish android builds to my Gitea instance, but it does not support ** glob pattern. 

### DETAILS
When I set the value of files to anything with **, it doesn't match the expected files even though ** is a standard glob pattern.

**Example**: the file `android/app/build/outputs/bundle/experimentalPlayDebug/app-experimental-play-debug.aab` cannot be matched with `android/app/build/outputs/**/*.aab`.

The name of the android build files can be different depending on some conditions, so it's not feasible to hard-code it as the value of the `files` option

### POSSIBLE SOLUTION
After some digging, I discovered that `filepath.Glob` does not support ** [Link to issue](https://github.com/golang/go/issues/11862).  [mattn/go-zglob](https://github.com/mattn/go-zglob) was recommended instead. 

I would have loved to create a PR, but I have no experience in Golang
